src/languages/en.json

```
@@ -383,5 +383,6 @@
   "my_rsk_domain": "My RSK domain",
   "email": "Email",
   "link": "Link",
-  "whatsapp": "WhatsApp"
+  "whatsapp": "WhatsApp",
+  "transaction_receipt_failed": "Transaction failed because it did not complete. This is usually caused by not enough gas being sent."
```

To state _because it did not complete_ we should also check by comparing the gas used and the gas limit. I will add this response improvement to another issue.

What about adding some link here now? Like

```suggestion
  "transaction_receipt_failed": "Transaction failed. Reach us out via $Gitter$ for support."
```

---

Sure. The error component can't handle the link right now. Perhaps I can look at expanding the component to accept markdown and we can add it that way. For now, let's keep the string and add a link to Gitter in the footer under "developers". It should probably be there anyway.

---

Good! I opened https://github.com/rnsdomains/rns-manager-react/issues/301